templates/engage/devops-cicd-webinar.html

```diff
@@ -0,0 +1,56 @@
+{% extends "engage/base_engage.html" %}
+
+{% block meta_description %}A webinar to learn how to automate Continuous Integration, Continuous Delivery, and Continuous Deployment with Kubernetes{% endblock %}
+
+{% block title %}Getting Started Easily With DevOps Practices: CI/CD{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1zAv3ouUrzQcQDw2DSefaePXcMu35BAumIMpzqmIkycs/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--image is-deep js-takeover is-dark p-takeover">
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h1>Getting Started Easily With DevOps Practices: CI/CD</h1>
```

Comment:
```suggestion
      <h1>Get started with DevOps best practices: CI/CD</h1>
```

Please change, the other is not sentence case and hard to read